google/auth/transport/_mtls_helper.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Helper functions for getting mTLS cert and key, for internal use only."""
+
+import json
+import logging
+from os import path
+import subprocess
+
+CONTEXT_AWARE_METADATA_PATH = "~/.secureConnect/context_aware_metadata.json"
+_CERT_PROVIDER_COMMAND = "cert_provider_command"
+_CERTIFICATE_SURFFIX = b"-----END CERTIFICATE-----\n"
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def read_metadata_file(metadata_path):
```

Comment:
`_read_dca_metadata_file`

we don't want this as part of the public API for the transport package.

also, I think this should probably move to its own package (we called it "cert" for Go)

Reply:
changed the name. Let's just keep it here for now. We can refactor once everything (http and grpc) is done.

Reply:
Oh, I'm wrong, anyway - I was thinking about how Go handles packages (all files in a directory are a package). Which is different than Python, of course.

Because the code is in a file prefixed with `_`, I think that's enough for it to not show up in documentation. Just making sure we're not creating a public API that our customers will see/try to depend on.